src/vs/workbench/contrib/terminalContrib/suggest/browser/terminalSuggestAddon.ts

```diff
@@ -178,9 +181,10 @@ export class SuggestAddon extends Disposable implements ITerminalAddon, ISuggest
 					this._promptInputModelSubscriptions.value = combinedDisposable(
 						this._promptInputModel.onDidChangeInput(e => this._sync(e)),
 						this._promptInputModel.onDidFinishInput(() => {
+							this._sendTelemetryInfo();
```

Comment:
There can be multiple completions per input, can we track them all and send them once the command finishes so we know if the exit code was 0?

Reply:
ah, this event has exitCode, will go with that.